crates/alloy/Cargo.toml

```diff
@@ -12,24 +12,33 @@ repository.workspace = true
 exclude.workspace = true
 
 [dependencies]
-alloy-core = { workspace = true, default-features = false, features = ["std"] }
+# core
+alloy-core = { workspace = true, default-features = false, optional = false }
```

Comment:
`optional = false` is the default value and can be omitted

Reply:
fixed